applications/API/apps.py

```
@@ -0,0 +1,9 @@
+from django.apps import AppConfig
+
+
+class ApiConfig(AppConfig):
+    default_auto_field = "django.db.models.BigAutoField"
+    name = "applications.API"
+    
+    def ready(self):
+        from applications.API import schema_extenstions #noqa
```

Pylint (3.3.7)

[convention] Import outside toplevel (applications.API.schema_extenstions)

(C0415)